src/auth/passwordreset/redux/actions/postAction.jsx

```diff
@@ -0,0 +1,30 @@
+import { RESET_PASSWORD_SUCCESS, RESET_PASSWORD_FAILURE, RESET_PASSWORD_LOADING } from '../types';
```

Comment:
There's no jsx in this file, change the extension

Reply:
This has been resolved, thank you.